scrapy/commands/parse.py

```diff
@@ -7,7 +10,9 @@
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
 from scrapy.exceptions import UsageError
-from scrapy import log
+
+logger = logging.getLogger('scrapy')
```

Comment:
I don't quite like `[scrapy.requests]` because there is no scrapy.requests module/package; it could make things harder to debug - it is harder to find where the logs came from. But I'm not sure, maybe `__name__` in log messages is too long. Could you please paste some examples after you try it?